pom.xml

```diff
@@ -81,33 +81,6 @@
       * io.opentelemetry.instrumentation:opentelemetry-resources
```

Comment:
Can we remove this big comment too, it's no longer needed now that we're removing the semconv declaration.